app/helpers/quadicon_helper.rb

```diff
@@ -641,6 +699,15 @@ def render_single_quad_quadicon(item, options)
     output.collect(&:html_safe).join('').html_safe
   end
 
+  def img_for_health_state(item)
+    case item.health_state
+    when "Valid"    then "100/healthstate-normal.png"
```

Comment:
@MaysaMacedo this is changed in this PR: https://github.com/ManageIQ/manageiq-ui-classic/pull/1166 . The new path to this image is: `svg/healthstate-normal.svg`